.github/workflows/php.yml

```diff
@@ -37,5 +37,8 @@ jobs:
         if: steps.composer-cache.outputs.cache-hit != 'true'
         run: composer install --prefer-dist --no-progress --no-suggest
 
+      - name: Copy .env.example
+        run: cp .env.example .env
+
```

Comment:
Non-blocking, Question. Not sure why we need to generate a .env file. The application must default to use whatever environment variable is set with or without this file. This .env file is mostly used for development purposes.